docs/guide/usage/integrations/home_assistant.md

```diff
@@ -89,7 +89,24 @@ automation:
 ## Groups
 Groups discovery is supported for groups of lights, switches, locks and covers. For other types you have to manually create a config in the Home Assistant `configuration.yaml`.
 
-## Exposing switch as a light
+## Overriding discovery properties
+Any Home Assistant MQTT discovery property can be overridden on a device. Two examples are shown below. For a full and current list of discovery properties, see [the Home Assistant MQTT Discovery integration](https://www.home-assistant.io/docs/mqtt/discovery/) and [the Home Assistant extension](https://github.com/Koenkk/zigbee2mqtt/blob/03ba647dc6b5f299f8f3ab441712999fcb3a253e/lib/extension/homeassistant.ts) in the Zigbee2MQTT source code.
+
+### Changing `supported_color_modes`
+This is useful for switching light bulbs from reporting values from X/Y (which is mandatory in the ZCL specification) to reporting in hue / saturation (which is what Philips Hue bulbs report color in).
```

Comment:
> from reporting values from X/Y (which is mandatory in the ZCL specification) to reporting in hue / saturation (which is what Philips Hue bulbs report color in).

This is not true. Hue bulbs report in X/Y just fine, only when controlling them via HS they report HS.

Reply:
I added a specific mention of changing via HS, and how that is triggered by `move_hue` and `move_sat`.